app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/util/UrlUtils.java

```diff
@@ -3,15 +3,19 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
+import java.util.regex.Pattern;
+
 /**
  * Created by fgei on 5/23/17.
  */
 
 public class UrlUtils {
 
+    private static final String WEB_URL_REGEX = "^(https?:\\/\\/)?([\\da-z\\.-]+\\.[a-z\\.]{2,6}|(([\\d]+[.]){3}[\\d]+))([\\/:?=&#]{1}[\\da-z\\.-]+)*[\\/\\?]?$";
```

Comment:
yes, that's the same from iOS :)
I think we'll benefit from using the same validation, in the end it's the same across both platform and having just one implementation will allow us to find the same bugs and have the same fixes! I'll change and test mine too